### PR TITLE
Upd #175819

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/pull/175819

1. Your comment

A new pattern of PopAds uncaught by the current rules found on `https://dlhd.so/stream/stream-51.php` I used unique `popads-node` for the regex, but simply generalized the rest of rules as they are relatively safe even without `$header`.  Add `dlhd.so#@%#//scriptlet('abort-current-inline-script', 'atob', '/popundersPerIP[\s\S]*?Date[\s\S]*?getElementsByTagName[\s\S]*?insertBefore/')` to reproduce.

2. Screenshots

<details>

<summary>Screenshot 1:</summary>

pa

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/general_url.txt [hunk 1]
```diff
@@ -26,11 +26,12 @@
...
+/^https:\/\/www\.[a-z]{8,16}\.com\/(?:[A-Za-z]+\/)*(?:[_0-9A-Za-z]{1,20}[-.])*[_0-9A-Za-z]{1,20}\.js$/$script,third-party,match-case,header=popads-node
 /^https:\/\/www\.[a-z]{8,16}\.com\/(?:[A-Za-z]+\/)*(?:[_0-9A-Za-z]{1,20}[-.])*[_0-9A-Za-z]{1,20}\.js$/$script,third-party,match-case,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/*.css|$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com*/images/*.min.js|$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/css/$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/js/css/$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
+://www.*.com/*.css|$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com*/images/*.min.js|$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com/css/$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com/js/css/$script,third-party,header=link:/\/>;rel=preconnect/
```
Comment (Alex-302) [inline BaseFilter/sections/general_url.txt:34]: @Yuki2718 Does not look safe. Could you please show a value of `link`?

Comment (Yuki2718) [reply BaseFilter/sections/general_url.txt:34]: `<https://cqnqrcddbacr.com/>;rel=preconnect` for `https://www.cqnqrcddbacr.com/mfirebase.css`. These rules are mostly safe even without `$header`, the only known FP is Principal Financial Group https://github.com/uBlockOrigin/uAssets/issues/22815.

<details>
<summary>Screenshot</summary>

header

</details>

Comment (Alex-302) [reply BaseFilter/sections/general_url.txt:34]: Hmm. I see `Popads-Node` header on the screenshot. What about to use it?
`link:****;rel=preconnect` is used also for legal purposes.

Comment (Yuki2718) [reply BaseFilter/sections/general_url.txt:34]: The old pattern without `Popads-Node` is still in use. So if we use this, we have to double up the number of rules.

Comment (Yuki2718) [reply BaseFilter/sections/general_url.txt:34]: Okay.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Thanks.

**Pull request merged**